core/src/main/java/fj/Ord.java

```diff
@@ -514,4 +514,21 @@ public int compare(A o1, A o2) {
   public Comparator<A> toComparator() {
 	  return new OrdComparator();
   }
+
+  /**
+   * Given two Ord instances that compare the given type of object, chain them together so
+   * that the result of the first is returned if it is not EQ, otherwise the result
+   * of the second is returned.  This can be used to reuse an Ord between subclasses
+   * or implementations of an interface.
+   *
+   * @return An Ord instance combining the two provided Ord instances
+   */
+  public static <A> Ord<A> chain(Ord<? super A> ord1, Ord<? super A> ord2) {
```

Comment:
You cannot 'directly' reuse it but you can still use ```comap``` which should do what you want at a small cost:
```java
    Ord<Date> dateOrd = Ord.comparableOrd();
    Ord<Timestamp> timestampOrd = dateOrd.comap(t -> t);
```
functionaljava does not support variance annotation because composition is usually preferred (over inheritance) when functional-programming.